app/templates/components/public/ticket-list.hbs

```diff
@@ -16,6 +16,9 @@
             <td>
               <div class="ui small">
                 {{ticket.name}}
+                {{#if (eq ticket.type 'donation')}}
+                  {{ui-popup tagName="i" class="info icon" content=(concat "Donation value should be between than " ticket.minPrice " " eventCurrency " and " ticket.maxPrice " " eventCurrency)}}
```

Comment:
The error message must come as a part of the input using the input error classes supported by semantic-ui

Reply:
@niranjan94 Done. But I've left the  `{{ui-popup}}` for information purposes. Is that alright?